src/miner.cpp

```diff
@@ -53,6 +53,8 @@ void RegenerateCommitments(CBlock& block, BlockManager& blockman)
 BlockAssembler::Options::Options() {
     blockMinFeeRate = CFeeRate(DEFAULT_BLOCK_MIN_TX_FEE);
     nBlockMaxWeight = DEFAULT_BLOCK_MAX_WEIGHT;
+    m_skip_inclusion_until = std::chrono::microseconds::max();
+    check_block_validity = true;
```

Comment:
Prefer default initialization in the declaration, rather than setting these in the ctor.